stellar/deploy-contract.js

```diff
@@ -155,20 +143,21 @@ function main() {
                 .default(15)
                 .argParser(Number),
         )
-        .action((options) => {
-            mainProcessor(options, processDeploy);
+        .action((contractName, options) => {
+            printInfo('Deploying contract', contractName);
```

Comment:
avoid printing in CLI setup. Deploy will print as part of the prompt it shows

Reply:
updated.